tempesta_fw/cfg.c

```diff
@@ -1368,13 +1368,9 @@ mod_stop(TfwCfgMod *mod)
 static void
 print_parse_error(const TfwCfgParserState *ps)
 {
-	const char *start = max((ps->pos - 80), ps->in);
-	int len = ps->pos - start;
+	long unsigned int len = ps->pos - ps->in;
 
-	TFW_ERR("configuration parsing error:\n"
-		"%.*s\n"
-		"^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
-		len, start);
+	TFW_ERR("configuration parsing error:%lu;%s\n", len, ps->e.name);
```

Comment:
```len``` is the total configuration file length - it doesn't help to locate the syntax error at all. Can we print line number instead?

Please add space after colon.

Next bad thing is that dmesg is full of warning messages:

        [  628.235453] [tempesta] ERROR: don't know how to handle: 'adfdffa'
        [  628.236993] ------------[ cut here ]------------
        [  628.238220] WARNING: CPU: 0 PID: 4495 at /root/tempesta/tempesta_fw/cfg.c:1425 tfw_cfg_parse_mods_cfg+0x10 8/0x1d0 [tempesta_fw]()
        ......
        [  628.306569] [tempesta] ERROR: configuration parsing error:43;adfdffa
        [  628.308057] ------------[ cut here ]------------
        [  628.309169] WARNING: CPU: 0 PID: 4495 at /root/tempesta/tempesta_fw/cfg.c:1373 tfw_cfg_parse_mods_cfg+0x14 2/0x1d0 [tempesta_fw]()
        ......
        [  628.364936] [tempesta] ERROR: can't parse configuration data
        [  628.366461] ------------[ cut here ]------------
        [  628.367817] WARNING: CPU: 0 PID: 4495 at /root/tempesta/tempesta_fw/cfg.c:1465 handle_sysctl_state_io+0x3d9/0x4a0 [tempesta_fw]()
        .........
        [  628.420853] [tempesta]     spec cleanup: 'server'
        [  628.422035] [tempesta] ERROR: failed to start modules
        [  628.423145] ------------[ cut here ]------------
        [  628.424612] WARNING: CPU: 0 PID: 4495 at /root/tempesta/tempesta_fw/cfg.c:1635 handle_sysctl_state_io+0x383/0x4a0 [tempesta_fw]()

Basically we have verbose enough error reporting and there is no need to the warnings with calltraces - they just pollute the log and make it hard to read.